### DISCUSSION TITLE
`omz changelog` does nothing

### DISCUSSION THREAD

**Discussion by rodhash** [ohmyzsh/ohmyzsh#12529] [Help] [answered]
This command was supposed to show latest changes applied, right?

Tried that cmd but it does nothing.. anyone else having this issue?

<img width="223" alt="image" src="https://github.com/ohmyzsh/ohmyzsh/assets/29671981/166295f5-5b98-42dc-9a20-b4ccfda888b5">

**Comment by carlosala** [comment, marked as answer]
It is expected, since last commit is dd4be1b6fb9973d63eba334d8bd92b3da30b3e72, which is a `chore` commit. `chore` commit do not show in changelog. Only [these kind](https://github.com/ohmyzsh/ohmyzsh/blob/dd4be1b6fb9973d63eba334d8bd92b3da30b3e72/tools/changelog.sh#L27-L32) of commits show.
